src/cmd/dut.rs

```diff
@@ -212,9 +213,23 @@ fn run_dut_monitor(args: &ArgsDutMonitor) -> Result<()> {
     let mut targets: Vec<MonitoredDut> = Vec::new();
     let mut port = 4022;
 
-    for dut in &args.duts {
-        targets.push(MonitoredDut::new(dut, port)?);
+    for raw_dut in &args.duts {
+        let ports: Vec<&str> = raw_dut.split(',').collect();
+        let mut fwports: Vec<PortForwarding> = vec![];
+        let dut_port = port;
+
         port += 1;
+
+        if ports.len() > 1 {
+            for fwport in ports.iter().skip(1) {
+                let parts: Vec<&str> = fwport.split(':').collect();
+                let port_no: u16 = parts[1].parse().expect("Invalid port number.");
```

Comment:
can you return the error with the string that coulddnt be parsed so whenever it is thrown that informatoin is included?

Reply:
Done.